File: cpp/Instruções Sequenciais/Q02.cpp
/*
------------------------------------------------
2. Crie um algoritmo/fluxograma/programa que 
leia 2 valores e escreva na tela a média 
ponderada entre eles. O primeiro valor tem peso 
40%, e o segundo valor tem peso 60%.
 */
#include <iostream>
#include <locale.h>
using namespace std;
int main () {
	setlocale(LC_ALL,"Portuguese");
	float a, b, media;
	cout<<"Informe o primeiro valor:";
	cin>>a;
	cout<<"Informe o segundo valor:";
	cin>>b;
	media = a*0.4 + b*0.6;
	cout<<"A média ponderada é:"<<media;
	return 0;
}

File: exercicios/python/Instruções Sequenciais/Q02.py
# Exercício 2 — Média ponderada (peso 40% e 60%)
def main():
    a = float(input("Informe o primeiro valor (peso 40%): "))
    b = float(input("Informe o segundo valor (peso 60%): "))
    media = a * 0.4 + b * 0.6
    print(f"Média ponderada: {media}")

if __name__ == "__main__":
    main()